docs/cimpl6UserGuide.md

```diff
@@ -69,35 +69,35 @@ As a best practice, CIMPL map files must have the *_map.txt* naming convention.
 
 Each map file should start with the following 3 lines:
 
-Grammar:	Map 6.0
-Namespace:  _data element group name_
-Target:		_fhir target version_
+* Grammar:	Map 6.0
+* Namespace:  _data element group name_
+* Target:		_fhir target version_
 
 Where:
 
-* _namespace name_ is the name of the grouping of elements that you have identified.
+* _data element group name_ is the name of the grouping of elements that you have identified.
 * _fhir target version_ is a choice of FHIR_DSTU_2, FHIR_STU_3, or FHIR_R4
 
 The snippet below shows an example:
 ```
-Grammar:	Map 5.1
+Grammar:	Map 6.0
```

Comment:
@GHCouderc - actually, it's Map 5.1, not 6.0.

Reply:
ok